ci/travis_script_python.sh

```diff
@@ -55,8 +52,12 @@ conda install -y -q \
       --file $TRAVIS_BUILD_DIR/ci/conda_env_python.yml \
       pip \
       numpy=1.13.1 \
+      python=${PYTHON_VERSION} \
```

Comment:
I had thought that having `python` in `conda_env_python.yml` was a bit weird

Reply:
That's actually a dependency of pyarrow :)
We can remove it though, python's version is pinned explicitly.

Reply:
The env file is for development and CI, right? It's not a conda recipe per se. I think it's better to remove Python from it, otherwise similar issues are bound to later happen.

Reply:
Done